girder/web_client/src/views/App.js

```diff
@@ -188,7 +189,11 @@ var App = View.extend({
         }
         this.$el.html(LayoutTemplate());
 
-        this.globalNavView.setElement(this.$('#g-global-nav-container')).render();
+        // Only show side navigation if logged in as an admin
+        if (getCurrentUser() && getCurrentUser().get('admin')) {
```

Comment:
@jeffbaumes We should remove the check for admin -- regular users need to be able to navigate the collections (and probably groups and users, too).